examples/BUILD.bazel

```diff
@@ -0,0 +1,29 @@
+alias(
```

Comment:
Are these aliases to ensure they are built as I don't see what in the examples depends on them. If so, could be worth adding a comment as to why they're here.

Reply:
It's more for convenience, referenced some of them in the readme for the chatter examples.

Reply:
Added docs.